org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+rename-icon: gaphor
+command: gaphor
+finish-args:
+- "--socket=x11"
+- "--socket=fallback-x11"
+- "--share=ipc"
+- "--socket=wayland"
+- "--persist=.gaphor"
+- "--filesystem=xdg-run/dconf"
+- "--filesystem=~/.config/dconfig:ro"
+- "--talk-name=ca.desrt.dconf"
+- "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+cleanup:
+- "/include"
+- "/lib/pkgconfig"
+- "/share/pkgconfig"
+- "/share/aclocal"
+- "/man"
+- "/share/man"
+- "/share/gtk-doc"
+- "*.la"
+- "*.a"
+- "/lib/girepository-1.0"
+- "/share/doc"
+- "/share/gir-1.0"
+modules:
+- name: gaphor
+  buildsystem: simple
+  build-options:
+    build-args:
+    - "--share=network"
```

Comment:
Builds are sandboxed so you can't access local filesystem or network. Take a look at [flatpak pip generator](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip).

Reply:
I submitted https://github.com/flatpak/flatpak-builder-tools/issues/50 since poetry is currently failing the pip-generator